erdpy/cli_contracts.py

```diff
@@ -169,13 +176,22 @@ def create(args: Any):
     projects.create_from_template(name, template, directory)
 
 
+def get_project_paths(args: Any) -> List[Path]:
+    base_path = Path(args.project)
+    recursive = bool(args.recursive)
+    if not recursive:
```

Comment:
Perhaps using the _positive_ condition would have been prettier.

Reply:
Done.